### PR TITLE
feat: add config property for showing taken hours on weekends in days view

### PR DESCRIPTION
**Pull request by mGasiorek998**

### CODE REVIEW

File: src/types/global.ts [hunk 1]
```diff
@@ -21,6 +21,13 @@ export type Config = {
   lang?: LangCodes;
   isFiltersButtonVisible?: boolean;
   maxRecordsPerPage?: number;
+  /**
+   * property for changing behavior of showing tooltip hours
+   * true - will always show 0 taken hours on weekends in day view
```
Comment (piotrNBTN) [inline src/types/global.ts:26]: The name of the property suggests that if it's true, the tooltip will show occupancy on weekends as well.

File: src/utils/getDayOccupancy.ts [hunk 2]
```diff
@@ -7,13 +7,15 @@ import { getTimeOccupancy } from "./getTimeOccupancy";
...
+  zoom: number,
+  includeFreeWeekends: boolean
 ): OccupancyData => {
   const focusedDayNum = focusedDate.isoWeekday();
   const getHoursAndMinutes: TimeUnits[] = occupancy.map((item) => {
     const { hours: itemHours, minutes: itemMinutes } = getDuration(item.occupancy);
 
-    if (focusedDayNum < 6) {
+    // if config was set to include free weekends max day num is 5 - friday else is 7 - whole week
+    if (focusedDayNum <= (includeFreeWeekends ? 5 : 7)) {
```
Comment (piotrNBTN) [inline src/utils/getDayOccupancy.ts:18]: The name of the property `includeFreeWeekends` suggests that if it's true, the tooltip will show occupancy on weekends as well.

**Review by piotrNBTN — CHANGES_REQUESTED** [review]

File: src/types/global.ts (hunk 1 above)
Comment (mGasiorek998) [reply src/types/global.ts]: Good catch, changed

File: src/utils/getDayOccupancy.ts (hunk 2 above)
Comment (mGasiorek998) [reply src/utils/getDayOccupancy.ts]: changed

**Review by piotrNBTN — APPROVED** [review]

**Pull request merged**